### ISSUE TITLE
Connect with more than one script to the CarMaker instance

### ISSUE THREAD

**Issue by flok** [gmnvh/pycarmaker#6]
Great work on this implementation.

Is it somehow possible to connect more than 1 Client to the CarMaker instance? I tried already to open more than one port but failed.

Thanks for the work

**Comment by gmnvh** [comment]
Hi. I am glad the code is helping you.

I think that it would be possible to create multiple instances since
carmaker windows are all clients, right?

What kind of error do you get?

**Comment by flok** [comment]
Yes it seems like its possible. But as its a normal TCP Connection you need to open multiple ports with the -cmdport command otherwise the Device refuses the connection.

Tried some ways of multiple -cmdport arguments or 1 -cmdport with multiple ports after it but couldnt get it to work.

Problem is basicly how we can open multiple ports.

**Comment by nag3939** [comment]
@flok , i am also facing with same issue ,do you have any idea
